dashboard/src/pages/api/run.ts

```diff
@@ -114,10 +120,12 @@ async function runSql(sql: string, dbAlias: string, username: string, password:
         app: 'run',
         env: process.env.IASQL_ENV,
         meta: {
+          sql,
           error: e.message,
           stack: e.stack,
         },
       });
+      res.status(500).json({ error: `Connection interruption while executing query ${sql}` });
```

Comment:
If we're going to hide the actual message it may make sense to generate a UUID and include it in both the `logger.error` and this JSON blob so if we trigger the error while running locally we can check the mezmo logs for that exact UUID and get the details.